server/tests/metadata.rs

```diff
@@ -103,4 +103,128 @@ mod entities_metadata_query {
             .await
             .should_be_bad_request();
     }
+
+    #[actix_rt::test]
+    async fn checks_mandatory_keys_in_response() {
+        Message::new("EntitiesMetadataQuery", json!({ "first": 350 }))
```

Comment:
No blocker but a follow up: I would make a fetch for an array of UUIDs each being of a different type we want support in the metadata API. Maybe a request like `returns_metadata_of_entities` but with different types is sufficient.